srcpkgs/ckb-next/files/ckb-next/log/run

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec logger -t ckb-next -p daemon.notice
```

Comment:
Shouldn't we be using the new vlogger ?

cc @Duncaen

Reply:
So i spoke to @Duncaen on IRC and he doesn't have an opinion on this. This package would be the first using vlogger I guess.

I'm unsure how to go from here. Should I just put vlogger instead of logger and we're done, is it ok as-is or...?

Reply:
just keep logger, if it's not a positive it is automatically a negative